vpc/tool/container2/setup_container_linux.go

```diff
@@ -406,6 +398,25 @@ out:
 		}
 	}
 
+	oldLinkRoute := netlink.Route{
```

Comment:
Can you add a comment, why does this route exist in the first place and why do we need to delete it?